PointerParameters.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+#define Size 10
+
+int sum(int *start, int *end)
+{
+   int initialization = 0;
+   while(start < end)
+   {
+      initialization += *start;
+   }
+   return initialization;
+}
+
+int main(void)
+{
+
+int Array[Size]= {20,10,5,39,4,16,19,26,31,20};
```

Comment:
```suggestion
int array[Size]= {20,10,5,39,4,16,19,26,31,20};
```

Some of your variables start with a capital letter and some start with a lowercase letter. It's very important to be consistent to make your code easier to read.